src/components/Frame/components/Toast/Toast.scss

```
@@ -8,10 +8,13 @@ $Backdrop-opacity: 0.88;
   display: inline-flex;
   max-width: rem(500px);
   padding: spacing(tight) spacing();
-  border-radius: border-radius();
-  background: rgba(color('black'), $Backdrop-opacity);
+  border-radius: var(--p-border-radius-wide, border-radius());
+  background: var(
+    --p-surface-inverse-foreground,
+    rgba(color('black'), $Backdrop-opacity)
+  );
   box-shadow: shadow();
```

Looks like toast now has no shadow so we can turn off like so:

```suggestion
  box-shadow: var(--p-none, shadow());
```

Override values are created in the overrides function in utils.ts.